src/test/java/com/osiris/autoplug/client/tasks/SSHServerTest.java

```diff
@@ -0,0 +1,139 @@
+/*
```

Comment:
Nice! Where you able to run this test and see if it works?

Reply:
no, I explain in this comment that  I was unable to run tests because I do not know how they work
https://github.com/Osiris-Team/AutoPlug-Client/pull/252#issuecomment-2190955648
If you could explain to me how to run and implement the tests, I very well could. Right now, I can only hope they work

Reply:
In intellij there is a green run button directly in the test class to run it, not sure how it's in vscode, Google run vscode Java JUnit test

Reply:
foregoing testing 😔